conftest.py

```
@@ -639,6 +644,102 @@ def destination_ocp_provider(fixture_store, destination_ocp_secret, ocp_admin_cl
+
+    yield {
+        "node_name": target_node,
+        "label_key": label_key,
+        "label_value": label_value,
+    }
+
+    # Cleanup: Remove the label
+    cleanup_node_label(ocp_admin_client, target_node, label_key)
+
+
+@pytest.fixture(scope="function")
```

Ruff (0.14.11)

`scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)